numpy-stubs/__init__.pyi

```diff
@@ -309,6 +309,8 @@ class ndarray(Generic[_DType]):
     def __add__(self, value: ndarray[_DType]) -> ndarray[_DType]: ...
     @overload
     def __add__(self, value: _DType) -> ndarray[_DType]: ...
+    @overload
+    def __add__(self, value: _FloatLike) -> ndarray[_DType]: ...
```

Comment:
what was the code where mypy complained?

Reply:
`np.random.randn(samples // 2) + 0.5`

```
ethicml/data/csvs/make_toy_dataset.py:15: error: No overload variant of "__add__" of "ndarray" matches argument type "float"  [operator]
ethicml/data/csvs/make_toy_dataset.py:15: note: Possible overload variants:
ethicml/data/csvs/make_toy_dataset.py:15: note:     def __add__(self, ndarray[float16]) -> ndarray[float16]
ethicml/data/csvs/make_toy_dataset.py:15: note:     def __add__(self, float16) -> ndarray[float16]
```

Reply:
👍